SeaPublicWebsite.UnitTests/QuestionFlowServiceTests.cs

```diff
@@ -161,25 +161,25 @@ public void RunSkipLinkTestCases(QuestionFlowServiceTestCase testCase)
                 new { reference = "ABCDEFGH" }
             )),
         new(
-            "Property type goes back to Postcode",
+            "Property type goes back to find EPC",
```

Comment:
I think so, we could take them back to postcode in that case but we're skipping the address and epc confirmation pages anyway in that case so I think it makes more sense to take them back to the actual start of the process. Happy to change if you disagree, would just need some more data management to make sure the epc was deleted if they went back to postcode and then clicked "cancel and continue without EPC"

Reply:
It could, though ConfirmEpcDetails would then go back to ConfirmAddressDetails only if there was 1 EPC found for the address, otherwise it would have to go back to FindEpc anyway as we don't have the postcode for which to fetch the multiple EPCs (and we'd have to set the single one differently as we can't fetch using the postcode so we'd have to manually add it somehow), right?

Reply:
I think ConfirmEpcDetails could always go back to FindEpc (I was a bit surprised it doesn't already)

Reply:
have tweaked the flow so it favours going back one step at a time where possible - if we want to not save postcode later on this may need updating but for now it is effective